packaging/generic-rpm/amazon-ecs-volume-plugin.socket

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Q. should it be updated to Copyright `2022` Amazon.com, Inc for
* packaging/generic-rpm/amazon-ecs-volume-plugin.service
* packaging/generic-rpm/amazon-ecs-volume-plugin.socket
* packaging/generic-deb/debian/amazon-ecs-volume-plugin.service
* packaging/generic-deb/debian/amazon-ecs-volume-plugin.socket ? 
Thanks.

Reply:
Q. should we reuse existing amazon-ecs-volume-plugin.service and amazon-ecs-volume-plugin.socket in the `amazon-linux-ami` folder instead of creating new files? Thanks.

Reply:
> should we reuse existing amazon-ecs-volume-plugin.service and amazon-ecs-volume-plugin.socket in the amazon-linux-ami folder instead of creating new files?

I see that all the packaging folders have their own ecs.service file as well, I'm not sure if the reason behind might this was to make the packaging folders to be self contained but  ya I do not have a preference in this.

Reply:
Updating to 2022. For the existing files I my thought is same as mythri's idea to have each packaging folders self contained and could be separated easily if we want to have different build root. And in case we have different setups for different platforms we gain more flexibilities on it.